README.md

```diff
@@ -1,30 +1,18 @@
-# HW_0
-## Homework 0: Make a README.md about yourself!
-Tell me a little bit about yourself and your interests.  
-Please make use of as many of the `.md` formatting tools as possible.  At a minimum, include:
-
-#### Multiple Headers
-##### And Header Styles
-----------------------  
-
-#### Lists  
-   1) Numbered lists
-      * bulleted lists
-          + more lists  
-
-#### Emphasis
-_Emphasis_  
-__Strong emphasis__   
-
-#### Code 
-In `line` code  
-
-```bash
-blocks of code  
-with multiple lines  
-```
-#### [Links to websites](https://amyhessl.faculty.wvu.edu/home)
-
-_Challenge:_ Add an image
+# A brief info about Tobi Ore
+------------------------------  
+## Background
+__Tobi Ore__, from Ekiti State, Nigeria, is pursuing a masters in geology at West Virginia University. He graduated from the University of Lagos in Nigeria with a bachelor’s degree in geophysics. 
+## Hobbies
+In his spare time, 
+   * he plays basketball (formerly professional); 
```

Comment:
Wow so cool! Can't wait to tell my son - he loves basketball.  Do you ever play at the rec center?

Reply:
Yes, I do play at the rec some weekends but mostly at my apartment every evening. I had a bad knee due to an accident, so I was dropped from the team in college.